packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1572,6 +1572,28 @@ Scenario that returns a different file encoding depending on the accept header.
 - image/png return a png image
 - image/jpeg return a jpeg image
 
+### Payload_MultiPart_FormData_basic
+
+- Endpoint: `post /multipart/form-data/mixed-parts`
+
+Expect request:
+
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+
+123
+--abcde12345
+Content-Disposition: form-data; name="profileImage ";
```

Comment:
fix the extra space in name

does request need a `filename=`?

Reply:
As for content-type, see https://github.com/Azure/cadl-ranch/pull/452#discussion_r1401715456

Reply:
![image](https://github.com/Azure/cadl-ranch/assets/53292327/995722dd-05a4-4c66-8e9e-781466d6b49d)

curl https://curl.se/docs/manpage.html